src/coreclr/jit/llvm.cpp

```diff
@@ -2607,10 +2689,31 @@ void Llvm::lowerCallToShadowStack(GenTreeCall* callNode)
     }
 }
 
+bool Llvm::independentCanBeReplacedWithItsField(LclVarDsc* addrVarDsc)
+{
+    return _compiler->lvaGetPromotionType(addrVarDsc) == Compiler::PROMOTION_TYPE_INDEPENDENT &&
+           addrVarDsc->CanBeReplacedWithItsField(_compiler);
+}
```

Comment:
`CanBeReplacedWithItsField` already checks for independent promotion so `independentCanBeReplacedWithItsField` is not needed.